lib/rucio/db/sqla/types.py

```diff
@@ -46,8 +45,11 @@ class GUID(TypeDecorator):
     otherwise uses CHAR(32), storing as stringified hex values.
 
     """
+
     impl = CHAR
 
+    cache_ok = True
```

Comment:
what and where is this used for?

Reply:
That's a new SQLAlchemy feature: https://docs.sqlalchemy.org/en/14/core/custom_types.html#sqlalchemy.types.TypeDecorator.cache_ok